packages/poaps/src/types/input.ts

```diff
@@ -0,0 +1,13 @@
+export interface FetchPoapsInput {
+  limit: number;
+  offset: number;
+  order?: string;
```

Comment:
What can be the values here? Would it throw an error on invalid order or just return empty response?